hedera-node/configuration/mainnet/bootstrap.properties

```diff
@@ -0,0 +1,2 @@
+# This file needs to exist whether or not there are any properties in it.
```

Comment:
```suggestion
# This file needs to exist even though there are any properties in it. Do NOT put properties in this file.
```

Reply:
We need a "why" in this. Somehow it got removed.

Reply:
Nvm, I needed to scroll down. It is there.